scripts/end2end_examples.py

```diff
@@ -0,0 +1,119 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+To run this script you need to install papermill apart from argilla:
+
+$ pip install papermill
+
+Then run the script:
+
+$ python scripts/end2end_examples.py --help
+"""
+
+import os
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Dict, Optional
+
+import papermill
+import typer
+
+EXAMPLES_FOLDER = (
+    Path(__file__).parent.parent / "docs/_source/tutorials_and_integrations/tutorials/feedback/end2end_examples"
+)
```

Comment:
I would say to define this as script argument:

```bash
end2end_examples "docs/_source/tutorials_and_integrations/tutorials/feedback/end2end_examples"
```

Reply:
I'll tackle this

Reply:
It has the following arguments now:

```bash
❯ python scripts/end2end_examples.py --help

 Usage: end2end_examples.py [OPTIONS]

 Run the end2end example notebooks. If no arguments are passed, it will try to get the api_key and the hf_token from the
 environment variables.

╭─ Options ─────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ --api-url                TEXT  [default: http://localhost:6900]                                                       │
│ --api-key                TEXT  [default: argilla.apikey]                                                              │
│ --hf-token               TEXT  [default: None]                                                                        │
│ --examples-folder        TEXT  [default: docs/_source/tutorials_and_integrations/tutorials/feedback/end2end_examples] │
│ --help                         Show this message and exit.                                                            │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

The expected path is set as default but can be overriden